apps/web/src/server/db/client.ts

```diff
@@ -9,9 +11,16 @@ declare global {
 
 export const prisma =
   global.prisma ||
-  new PrismaClient({
-    log: ["error"],
-  });
+  // enable planetscale adapter in production if enabled
+  (env.NODE_ENV === "production" && env.USE_PLANETSCALE
+    ? (() => {
+        const connection = connect({ url: env.DATABASE_URL });
+        const adapter = new PrismaPlanetScale(connection);
+        return new PrismaClient({ adapter });
```

Comment:
I think we should add the `log: ["error"]` here too. Else we would loose those in production with Planetscale, wouldn't we?